box-ui/src/components/ts/JitsiComponent.tsx

```diff
@@ -68,6 +79,19 @@ const instantiateJitsi = async (
 const JitsiMeetComponent: FunctionComponent<InputRoomProps> = (props: InputRoomProps) => {
     const [displayHangup, setDisplayHangup] = useState<boolean>();
     const navigate = useNavigate();
+    const [raised, setRaised] = useState(false);
+    const [idTo, setidTo] = useState();
+    function lowerHand() {
+        setRaised((raised) => (raised ? !raised : raised));
+    }
+    function switchState() {
+        console.log(raised);
+        setidTo((idTo) => idTo);
+        setRaised((raised) => !raised);
```

Comment:
```suggestion
        setRaised(!raised);
```

Reply:
In the official documentation for react, functional updates are required to set a new state based on the old state: https://reactjs.org/docs/hooks-reference.html#functional-updates , using a normal update doesn't change the state